templates/layout/region.html.twig

```diff
@@ -2,26 +2,15 @@
 /**
  * @file
  * Theme override to display a region.
- *
- * Available variables:
- * - content: The content for this region, typically blocks.
- * - attributes: HTML attributes for the region div.
- * - region: The name of the region variable as defined in the theme's
- *   .info.yml file.
- *
- * @see template_preprocess_region()
```

Comment:
I don't have any particularly strong opinions on it, but let's create an issue to discuss?